<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igwSpreadSheet</class>
 <widget class="QWidget" name="igwSpreadSheet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_Precision">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>精度:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxPrecision">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ExportSpreadsheet">
       <property name="icon">
        <iconset resource="../res.qrc">
         <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwSaveCSV.png</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwSaveCSV.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="actionExportSpreadSheet">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwSaveCSV.png</normaloff>:/igwWidgets/Icons/Widgets/Resources/Icons/igwSaveCSV.png</iconset>
   </property>
   <property name="text">
    <string>ExportSpreadSheet</string>
   </property>
   <property name="toolTip">
    <string>导出文件为CSV或者txt文件</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
